starter-code/index.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="../starter-code/css/main.css" />
```

Comment:
Make sure to link your own CSS file AFTER any libraries such as Bootstrap. What is happening here is that the bootstrap css is overriding your own, causing unexpected results.